Cargo.toml

```diff
@@ -22,18 +22,20 @@ libc = "0.2.104"
 log = {version = "0.4.17", optional = true }
 pyo3 = {version = "0.18.0", features=["extension-module", "abi3-py37"], optional = true }
 
-[dev-dependencies]
+[target.'cfg(not(target_os = "wasi"))'.dev-dependencies]
```

Comment:
I *really* hate this, but I don't think there is any other way to do benchmark specific dependencies (even with features). If you have any ideas though, I am all ears.

(This divides test specific deps and benchmark specific deps, since essentially none of the benchmark deps compile to WASI)